portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+  <html>
+<head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    </head>
+  <body>
+    <nav>
```

Comment:
It's unfortunate that this nav header has to be duplicated on every page right now, but I'm sure there will be instruction in the upcoming weeks on how to make this dynamic with server-side templating.